site/js/editor.js

```diff
@@ -209,11 +209,13 @@ NFAEditor.prototype = {
         }
         return true;
     },
-    gotoStep: function( input, step ) {
+    gotoStep: function( input, steps ) {
```

Comment:
Why are you renaming this? The semantics as I understand them are: `gotoStep( input, step )` means to use input `input` and go to step number `step`. So, since it is the index of the particular step that you want to reach, it should be named `step`. I can see that you may consider it to mean that the algorithm will step through `steps` steps, and this makes sense, but the name of the function should then be something else.